src/dns_operations/mod.rs

```diff
@@ -225,6 +229,7 @@ impl DnsOperations {
     }
 
     fn get_housing_structured_data(&self, long_name: &String) -> Result<::routing::structured_data::StructuredData, ::errors::DnsError> {
+        debug!("Retrieving housing structured data ...");
```

Comment:
For learning purposes @ustulation could you briefly tell me why this is unnecessary?